app/design/frontend/Diana/default-theme/Magento_Theme/web/js/cart-user-bar.js

```diff
@@ -0,0 +1,10 @@
+define(['jquery'], function($) {
+    'use strict';
+
+    return function() {
```

Comment:
@Dianaeva Please, create 1 JS file with functionality for actions and pass selectors as parameters